village/src/components/SmurfForm.js

```diff
@@ -13,7 +13,13 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
-
+    axios
+    .post("http://localhost:3333/smurfs", this.state)
+    .then(response => {
+      console.log(response);
+      window.location.reload();
```

Comment:
Knowing what you do now, do you think `window.location.reload()` is good practice?